src/App.jsx

```diff
@@ -12,6 +12,9 @@ import MainLayout from '@/layouts/MainLayout';
 import AppLayout from '@/layouts/AppLayout';
 import { RegisterStaff } from '@/pages/app/RegisterStaff';
 import { AuthProvider, ProtectedRoute } from '@/components/auth-provider';
+import { ClinicCreation } from '@/pages/ClinicCreation';
+import  {ClinicaEdicion} from '@/pages/EditClinic';
+import {ClinicaCompletada} from '@/pages/SuccessPage';
```

Comment:
this won't import correctly

```suggestion
import { ClinicCreation } from '@/pages/app/ClinicCreation';
import { ClinicaEdicion } from '@/pages/app/EditClinic';
import { ClinicaCompletada } from '@/pages/app/SuccessPage';
```